appinventor/docs/markdown/reference/components/media.md

```diff
@@ -381,7 +381,9 @@ None
 
 {:id="SpeechRecognizer.UseLegacy" .boolean} *UseLegacy*
 : If true, a separate dialog is used to recognize speech (the default). If false, speech is
- recognized in the background and partial results are also provided. See
+recognized in the background and updates are received as it recognizes words. [`AfterGettingText`](#SpeechRecognizer.AfterGettingText) may get several calls with `partial` set to  `false`. Once sufficient
+time has elapsed since the last utterance, or `StopListening` is called, the last string will be returned with  set `partial` to `true` to indicate that it is the
```

Comment:
@ellelili2025 This is backward. "the last string will be returned with `partial` set to `false` to indicate...